README.md

```diff
@@ -39,7 +39,30 @@ pipenv shell
 python3 bot.py
 ```
 
-### Tokens
+### QRcode Tokens
+
+#### How to Get These Tokens?
+
+```sql
+SELECT
+  REPLACE(JSON_EXTRACT(ATTENDEE_INFO, '$.qrcode'), '"', '') AS token
+FROM
+  `pycontw-225217.ods.ods_kktix_attendeeId_datetime`
+WHERE
+  (REFUNDED IS NULL
+    OR REFUNDED = FALSE)
+  AND NAME LIKE '%<year>%'  -- year param: 20xx of course
+  AND LOWER(NAME) LIKE '%<type>%';  -- type param: <corporate, individual, reserved>
+```
```

Comment:
if u get a chance, would you like to invoke these SQLs and get the qrcode tokens by yourself?

I think it might be better to have multiple volunteers (you and me) knowing how to deploy/invoke `registration bot` E2E in PyConTW community, thoughts?

Reply:
@david30907d Against where should I invoke these sql? Is this on metabase?

Reply:
bigquery or metabase/ods dataset.
FYI: I've integrated web team's pg and datateam's bigquery to metabase